### ISSUE TITLE
FR: Change Font CSS 

### ISSUE THREAD

**Issue by ZoeBeykirch** [aemsites/vg-macktrucks-com#66]
### Description

To comply with legal requirements, the font "Helvetica" and fallback font "Arial" owned by Monotype, must be replaced. "[Inter](https://fonts.google.com/specimen/Inter/license)" an open-source font, will serve as the new CSS fallback font across all Mack Truck website domains.

### Priority

Critical

### Acceptance Criteria 

Some identified areas where Arial exists (TBU): @alexiscoelho 
- [x] Dealer locator 
- [x] HTML pages fallback 
- [x]  Remove all fallback font information related to “Arial” and "Helvetica" and change it to "[Inter](https://fonts.google.com/specimen/Inter/license)" for all Mack Trucks domains:
     - [x] [USA](https://www.macktrucks.com/)
     - [x] [Canada](https://www.macktrucks.ca/)
     - [x] [Mexico](https://www.macktrucks.com.mx/)
     - [x] [Australia](https://www.macktrucks.com.au/)
     - [x] [New Zealand](https://www.macktrucks.co.nz/)
     - [x] [Bahamas](https://bahamas.macktrucks.com/)
     - [x] [Dominican Republic](https://www.macktrucks.com.do/)
     - [x] [Haiti](https://www.macktrucks.ht/)
     - [x] [Costa Rica](https://www.macktrucks.cr/)
     - [x] [El Salvador](https://sv.macktrucks.com/)
     - [x] [Guatemala](https://www.macktrucks.gt/)
     - [x] [Panama](https://www.macktrucks.com.pa/)
     - [x] [Nicaragua](https://www.macktrucksnicaragua.com/)
     - [x] [Honduras](https://www.macktrucks.hn/)
     - [x] [Middle East](https://www.macktrucksemea.com/)
     - [x] [Nigeria](https://www.macktrucksnigeria.com/)
     - [x] [Argentina](https://www.macktrucks.com.ar/)
     - [x] [Bolivia](https://www.macktrucks.com.bo/)
     - [x] [Chile](https://www.macktrucks.cl/)
     - [x] [Colombia](https://www.macktrucks.com.co/)
     - [x] [Ecuador](https://www.macktrucks.com.ec/)
     - [x] [Guyana](https://www.macktrucksguyana.com/)
     - [x] [Peru](https://www.macktrucks.com.pe/)
     - [x] [Trinidad and Tobago](https://tt.macktrucks.com/)
     - [x] [Venezuela](https://www.macktrucks.com.ve/)

- [ ] Once the above is completed, we should include a link to a license notice on the relevant websites so that we are in line with good open source practice. (see e.g. https://www.lego.com/en-us/legal/notices-and-policies/open-source/?locale=en-us as an example, where LEGO provides a notice for its use of Noto Sarif under the OFL) -**- To be handled by CMS team outside of this ticket.** 

- [x]  Verify font usage across all sites for layout issues, ideally in a staging environment prior to go-live.

### Value creation

Compliance & standards

### Cost of fixing

Low cost

### Technical complexity

S

### References

Inter (by Google Inc.) as a fallback font for websites: https://fonts.google.com/specimen/Inter/license
• Inter is licensed under the SIL Open Font Licence V. 1.1 (“OFL”)

### Out of Scope

- [x] Helvetica to be addressed in a separate ticket when needed

### Related Tickets
https://github.com/hlxsites/vg-roadchoice-com/issues/251
https://github.com/hlxsites/vg-volvotrucks-us/issues/667
https://github.com/hlxsites/vg-partsasist/issues/114

**Comment by jonatan-lledo-netcentric** [comment]
regarding Helvetica, this also apply for _Helvetica Neue LT Pro_ and _Helvetica Neue_?
the following font family list are affected:
- _Helvetica Neue LT Pro 55 Roman_
- _Helvetica Neue LT Pro 65 Medium_
- _Helvetica Neue LT Pro 75 Bold_
- _Helvetica Neue LT Pro 56 Roman Italic_
- _Helvetica Neue LT Pro 66 Medium Italic_
- _Helvetica Neue LT Pro 76 Bold Italic_
- _Helvetica Neue 85 Heavy_

**Comment by ZoeBeykirch** [comment]
@jonatan-lledo-netcentric Yes - all of those related fonts for Helvetica will need to be removed and replaced.

**Comment by silviulungu** [comment]
Testing completed. Everything works accordingly!

**Comment by jonatan-lledo-netcentric** [comment]
Helvetica is also addressed in this ticket.
The 2 remaining ACs, are open to be taken care by the content team.

**Comment by silviulungu** [comment]
Testing finished on one more market (Bahamas). Everything works accordingly.

**Comment by BeckyMedlin** [comment]
@Meli-lazarte  and @Kannan-CM Moving content task to CMS board in Sharepoint: Once the above is completed, we should include a link to a license notice on the relevant websites so that we are in line with good open source practice. (see e.g. https://www.lego.com/en-us/legal/notices-and-policies/open-source/?locale=en-us as an example, where LEGO provides a notice for its use of Noto Sarif under the OFL)

**Comment by jonatan-lledo-netcentric** [comment]
Since the task of creating the content page referencing the source licenses is being handled in another task, I consider this one closed.

**Comment by bouchriy** [comment]
This issue has been migrated to Azure DevOps: https://dev.azure.com/volvo-digital-experience/DEX Platform/_workitems/edit/132234
